practice-one/exercise-2/part-9/js/script.js

```diff
@@ -0,0 +1,14 @@
+const getOptions = () => {
+  const colorSelection = document.getElementById('colorList');
+  txt = 'No. of items in dropdown is: ';
+  let colorList = document.getElementById('colorList').length;
```

Comment:
variable name does not make sense. Btw, it does not change, so you should use const. And after define variable should have empty line